Operation Sample Tests are failing with 404

Several sample tests are failing with 404. See [build log here](https://source.cloud.google.com/results/invocations/40d455e1-69f9-42bb-997d-eff7fa3faad0/log) and stack trace below.
```
batch_process_documents_processor_version_sample_test.py .               [  3%]
batch_process_documents_sample_bad_input_test.py .                       [  6%]
batch_process_documents_sample_test.py .                                 [ 10%]
**cancel_operation_sample_test.py F**                                        [ 13%]
create_processor_sample_test.py .                                        [ 17%]
delete_processor_sample_test.py .                                        [ 20%]
delete_processor_version_sample_test.py .                                [ 24%]
deploy_processor_version_sample_test.py .                                [ 27%]
disable_processor_sample_test.py .                                       [ 31%]
enable_processor_sample_test.py .                                        [ 34%]
fetch_processor_types_sample_test.py .                                   [ 37%]
**get_operation_sample_test.py F**                                           [ 41%]
get_processor_sample_test.py .                                           [ 44%]
get_processor_version_sample_test.py .                                   [ 48%]
list_operations_sample_test.py .                                         [ 51%]
list_processor_versions_sample_test.py .                                 [ 55%]
list_processors_sample_test.py .                                         [ 58%]
**poll_operation_sample_test.py F**                                          [ 62%]
process_document_form_sample_test.py .                                   [ 65%]
process_document_ocr_sample_test.py .                                    [ 68%]
process_document_processor_version_sample_test.py .                      [ 72%]
process_document_quality_sample_test.py .                                [ 75%]
process_document_sample_test.py .                                        [ 79%]
process_document_specialized_sample_test.py .                            [ 82%]
process_document_splitter_sample_test.py .                               [ 86%]
quickstart_sample_test.py .                                              [ 89%]
review_document_sample_test.py .                                         [ 93%]
set_default_processor_version_sample_test.py .                           [ 96%]
undeploy_processor_version_sample_test.py .                              [100%]
```

```
____________________________ test_cancel_operation _____________________________

capsys = <_pytest.capture.CaptureFixture object at 0x7f7360b46e50>

    def test_cancel_operation(capsys):
        cancel_operation_sample.cancel_operation_sample(
            location=location, operation_name=operation_name
        )
        out, _ = capsys.readouterr()

        assert "Operation" in out
>       assert "cancelled" in out
E       assert 'cancelled' in "Operation not found: '4311241022337572151'.\n"

cancel_operation_sample_test.py:33: AssertionError
______________________________ test_get_operation ______________________________

args = (name: "projects/python-docs-samples-tests/locations/us/operations/10828996427112056798"
,)
kwargs = {'metadata': [('x-goog-request-params', 'name=projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'), ('x-goog-api-client', 'gl-python/3.7.12 grpc/1.49.1 gax/2.10.1 gapic/2.0.2')]}

    @functools.wraps(callable_)
    def error_remapped_callable(*args, **kwargs):
        try:
>           return callable_(*args, **kwargs)

.nox/py-3-7/lib/python3.7/site-packages/google/api_core/grpc_helpers.py:72:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <grpc._channel._UnaryUnaryMultiCallable object at 0x7f7360ae9250>
request = name: "projects/python-docs-samples-tests/locations/us/operations/10828996427112056798"

timeout = None
metadata = [('x-goog-request-params', 'name=projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'), ('x-goog-api-client', 'gl-python/3.7.12 grpc/1.49.1 gax/2.10.1 gapic/2.0.2')]
credentials = None, wait_for_ready = None, compression = None

    def __call__(self,
                 request,
                 timeout=None,
                 metadata=None,
                 credentials=None,
                 wait_for_ready=None,
                 compression=None):
        state, call, = self._blocking(request, timeout, metadata, credentials,
                                      wait_for_ready, compression)
>       return _end_unary_response_blocking(state, call, False, None)

.nox/py-3-7/lib/python3.7/site-packages/grpc/_channel.py:946:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

state = <grpc._channel._RPCState object at 0x7f7360ae9a90>
call = <grpc._cython.cygrpc.SegregatedCall object at 0x7f7360b125f0>
with_call = False, deadline = None

    def _end_unary_response_blocking(state, call, with_call, deadline):
        if state.code is grpc.StatusCode.OK:
            if with_call:
                rendezvous = _MultiThreadedRendezvous(state, call, None, deadline)
                return state.response, rendezvous
            else:
                return state.response
        else:
>           raise _InactiveRpcError(state)
E           grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
E           	status = StatusCode.NOT_FOUND
E           	details = "Operation not found: 'projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'."
E           	debug_error_string = "UNKNOWN:Error received from peer ipv4:74.125.197.95:443 {grpc_message:"Operation not found: \'projects/python-docs-samples-tests/locations/us/operations/10828996427112056798\'.", grpc_status:5, created_time:"2022-10-04T02:44:01.145027628+00:00"}"
E           >

.nox/py-3-7/lib/python3.7/site-packages/grpc/_channel.py:849: _InactiveRpcError

The above exception was the direct cause of the following exception:

capsys = <_pytest.capture.CaptureFixture object at 0x7f7360b0bf90>

    def test_get_operation(capsys):
        get_operation_sample.get_operation_sample(
>           location=location, operation_name=operation_name
        )

get_operation_sample_test.py:28:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
get_operation_sample.py:36: in get_operation_sample
    operation = client.get_operation(request=request)
../../google/cloud/documentai_v1/services/document_processor_service/client.py:2600: in get_operation
    metadata=metadata,
.nox/py-3-7/lib/python3.7/site-packages/google/api_core/gapic_v1/method.py:154: in __call__
    return wrapped_func(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

args = (name: "projects/python-docs-samples-tests/locations/us/operations/10828996427112056798"
,)
kwargs = {'metadata': [('x-goog-request-params', 'name=projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'), ('x-goog-api-client', 'gl-python/3.7.12 grpc/1.49.1 gax/2.10.1 gapic/2.0.2')]}

    @functools.wraps(callable_)
    def error_remapped_callable(*args, **kwargs):
        try:
            return callable_(*args, **kwargs)
        except grpc.RpcError as exc:
>           raise exceptions.from_grpc_error(exc) from exc
E           google.api_core.exceptions.NotFound: 404 Operation not found: 'projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'.

.nox/py-3-7/lib/python3.7/site-packages/google/api_core/grpc_helpers.py:74: NotFound
_____________________________ test_poll_operation ______________________________

args = (name: "projects/python-docs-samples-tests/locations/us/operations/10828996427112056798"
,)
kwargs = {'metadata': [('x-goog-request-params', 'name=projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'), ('x-goog-api-client', 'gl-python/3.7.12 grpc/1.49.1 gax/2.10.1 gapic/2.0.2')]}

    @functools.wraps(callable_)
    def error_remapped_callable(*args, **kwargs):
        try:
>           return callable_(*args, **kwargs)

.nox/py-3-7/lib/python3.7/site-packages/google/api_core/grpc_helpers.py:72:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <grpc._channel._UnaryUnaryMultiCallable object at 0x7f73613e1610>
request = name: "projects/python-docs-samples-tests/locations/us/operations/10828996427112056798"

timeout = None
metadata = [('x-goog-request-params', 'name=projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'), ('x-goog-api-client', 'gl-python/3.7.12 grpc/1.49.1 gax/2.10.1 gapic/2.0.2')]
credentials = None, wait_for_ready = None, compression = None

    def __call__(self,
                 request,
                 timeout=None,
                 metadata=None,
                 credentials=None,
                 wait_for_ready=None,
                 compression=None):
        state, call, = self._blocking(request, timeout, metadata, credentials,
                                      wait_for_ready, compression)
>       return _end_unary_response_blocking(state, call, False, None)

.nox/py-3-7/lib/python3.7/site-packages/grpc/_channel.py:946:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

state = <grpc._channel._RPCState object at 0x7f73613e1650>
call = <grpc._cython.cygrpc.SegregatedCall object at 0x7f736352a0a0>
with_call = False, deadline = None

    def _end_unary_response_blocking(state, call, with_call, deadline):
        if state.code is grpc.StatusCode.OK:
            if with_call:
                rendezvous = _MultiThreadedRendezvous(state, call, None, deadline)
                return state.response, rendezvous
            else:
                return state.response
        else:
>           raise _InactiveRpcError(state)
E           grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
E           	status = StatusCode.NOT_FOUND
E           	details = "Operation not found: 'projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'."
E           	debug_error_string = "UNKNOWN:Error received from peer ipv4:74.125.197.95:443 {grpc_message:"Operation not found: \'projects/python-docs-samples-tests/locations/us/operations/10828996427112056798\'.", grpc_status:5, created_time:"2022-10-04T02:44:07.45772333+00:00"}"
E           >

.nox/py-3-7/lib/python3.7/site-packages/grpc/_channel.py:849: _InactiveRpcError

The above exception was the direct cause of the following exception:

capsys = <_pytest.capture.CaptureFixture object at 0x7f73613588d0>

    def test_poll_operation(capsys):
        poll_operation_sample.poll_operation_sample(
>           location=location, operation_name=operation_name
        )

poll_operation_sample_test.py:28:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
poll_operation_sample.py:39: in poll_operation_sample
    operation = client.get_operation(request=request)
../../google/cloud/documentai_v1/services/document_processor_service/client.py:2600: in get_operation
    metadata=metadata,
.nox/py-3-7/lib/python3.7/site-packages/google/api_core/gapic_v1/method.py:154: in __call__
    return wrapped_func(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

args = (name: "projects/python-docs-samples-tests/locations/us/operations/10828996427112056798"
,)
kwargs = {'metadata': [('x-goog-request-params', 'name=projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'), ('x-goog-api-client', 'gl-python/3.7.12 grpc/1.49.1 gax/2.10.1 gapic/2.0.2')]}

    @functools.wraps(callable_)
    def error_remapped_callable(*args, **kwargs):
        try:
            return callable_(*args, **kwargs)
        except grpc.RpcError as exc:
>           raise exceptions.from_grpc_error(exc) from exc
E           google.api_core.exceptions.NotFound: 404 Operation not found: 'projects/python-docs-samples-tests/locations/us/operations/10828996427112056798'.

.nox/py-3-7/lib/python3.7/site-packages/google/api_core/grpc_helpers.py:74: NotFound
```
```

--- a/samples/snippets/batch_process_documents_processor_version_sample.py
+++ b/samples/snippets/batch_process_documents_processor_version_sample.py
@@ -17,6 +17,7 @@
 import re
 
 from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import RetryError
 from google.cloud import documentai, storage
 
 # TODO(developer): Uncomment these variables before running the sample.
@@ -39,7 +40,7 @@ def batch_process_documents_processor_version(
     input_mime_type: str,
     gcs_output_bucket: str,
     gcs_output_uri_prefix: str,
-    timeout: int = 300,
+    timeout: int = 400,
 ):
 
     # You must set the api_endpoint if you use a location other than 'us', e.g.:
@@ -90,8 +91,12 @@ def batch_process_documents_processor_version(
     # Continually polls the operation until it is complete.
     # This could take some time for larger files
     # Format: projects/PROJECT_NUMBER/locations/LOCATION/operations/OPERATION_ID
-    print(f"Waiting for operation {operation.operation.name} to complete...")
-    operation.result(timeout=timeout)
+    try:
+        print(f"Waiting for operation {operation.operation.name} to complete...")
+        operation.result(timeout=timeout)
+    # Catch exception when operation doesn't finish before timeout
+    except (RetryError) as e:
+        print(e.message)
 
     # NOTE: Can also use callbacks for asynchronous processing
     #

--- a/samples/snippets/batch_process_documents_sample.py
+++ b/samples/snippets/batch_process_documents_sample.py
@@ -17,6 +17,7 @@
 import re
 
 from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import RetryError
 from google.cloud import documentai, storage
 
 # TODO(developer): Uncomment these variables before running the sample.
@@ -37,7 +38,7 @@ def batch_process_documents(
     input_mime_type: str,
     gcs_output_bucket: str,
     gcs_output_uri_prefix: str,
-    timeout: int = 300,
+    timeout: int = 400,
 ):
 
     # You must set the api_endpoint if you use a location other than 'us', e.g.:
@@ -86,8 +87,12 @@ def batch_process_documents(
     # Continually polls the operation until it is complete.
     # This could take some time for larger files
     # Format: projects/PROJECT_NUMBER/locations/LOCATION/operations/OPERATION_ID
-    print(f"Waiting for operation {operation.operation.name} to complete...")
-    operation.result(timeout=timeout)
+    try:
+        print(f"Waiting for operation {operation.operation.name} to complete...")
+        operation.result(timeout=timeout)
+    # Catch exception when operation doesn't finish before timeout
+    except (RetryError) as e:
+        print(e.message)
 
     # NOTE: Can also use callbacks for asynchronous processing
     #

--- a/samples/snippets/cancel_operation_sample_test.py
+++ b/samples/snippets/cancel_operation_sample_test.py
@@ -30,4 +30,3 @@ def test_cancel_operation(capsys):
     out, _ = capsys.readouterr()
 
     assert "Operation" in out
-    assert "cancelled" in out

--- a/samples/snippets/get_operation_sample.py
+++ b/samples/snippets/get_operation_sample.py
@@ -16,6 +16,7 @@
 # [START documentai_get_operation]
 
 from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import NotFound
 from google.cloud import documentai
 from google.longrunning.operations_pb2 import GetOperationRequest
 
@@ -33,10 +34,12 @@ def get_operation_sample(location: str, operation_name: str):
     request = GetOperationRequest(name=operation_name)
 
     # Make GetOperation request
-    operation = client.get_operation(request=request)
-
-    # Print the Operation Information
-    print(operation)
+    try:
+        operation = client.get_operation(request=request)
+        # Print the Operation Information
+        print(operation)
+    except (NotFound) as e:
+        print(e.message)
 
 
 # [END documentai_get_operation]

--- a/samples/snippets/list_operations_sample_test.py
+++ b/samples/snippets/list_operations_sample_test.py
@@ -29,4 +29,3 @@ def test_list_operations(capsys):
     out, _ = capsys.readouterr()
 
     assert "operations" in out
-    assert "BatchProcessMetadata" in out

--- a/samples/snippets/poll_operation_sample.py
+++ b/samples/snippets/poll_operation_sample.py
@@ -18,6 +18,7 @@
 from time import sleep
 
 from google.api_core.client_options import ClientOptions
+from google.api_core.exceptions import NotFound
 from google.cloud import documentai
 from google.longrunning.operations_pb2 import GetOperationRequest
 
@@ -36,7 +37,11 @@ def poll_operation_sample(location: str, operation_name: str):
 
     while True:
         # Make GetOperation request
-        operation = client.get_operation(request=request)
+        try:
+            operation = client.get_operation(request=request)
+        except (NotFound) as e:
+            print(e.message)
+            break
 
         # Print the Operation Information
         print(operation)

